libraries/AP_Declination/tables.cpp

```diff
@@ -9,7 +9,7 @@ const float AP_Declination::SAMPLING_MAX_LAT = 90;
 const float AP_Declination::SAMPLING_MIN_LON = -180;
 const float AP_Declination::SAMPLING_MAX_LON = 180;
 
-const float AP_Declination::declination_table[LAT_TABLE_SIZE][LON_TABLE_SIZE] = {
+__EXTFLASHFUNC__ const float AP_Declination::declination_table[LAT_TABLE_SIZE][LON_TABLE_SIZE] = {
```

Comment:
Typicall data tables might go in a different area - do we know these are only accessed at startup?

Reply:
These aren't just used at startup.

But I believe they're used at very low rates - of the order of 1 or 2Hz.

And probably quite rapidly while doing mag field calibrations.

Reply:
I mean its probably ok, but I just can't say for sure - it depends what the usage looks like